frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -501,6 +510,32 @@ class FilesApi extends ResourceApi<UFile, ProductStorage, UFileSpecification,
                     );
                 }
             },
+            {
+                icon: "refresh",
+                text: "Synchronization",
+                enabled: (selected, cb) => {
+                    const support = cb.collection?.status.resolvedSupport?.support;
+                    return cb.embedded !== true &&
```

Comment:
Did you merge with `origin/storage` recently? I'm not sure `cb.embbeded` still exists.